dali/python/nvidia/dali/_multiproc/pool.py

```diff
@@ -202,10 +204,14 @@ def __init__(
             task_r, task_w = mp.Pipe(duplex=False)
             res_r, res_w = mp.Pipe(duplex=False)
             sock_reader, sock_writer = socket.socketpair()
+            if callback_pickler is None:
+                callbacks_ = callbacks
```

Comment:
How about this?
```suggestion
                callbacks_arg = callbacks
```
I'm not a fan of trailing underscore and leading underscore has a meaning in Python which I guess we should not abuse.

Reply:
done